### PR TITLE
Add 'poster' for MozFest video

### PR DESCRIPTION
**Pull request by mmmavis**
Show a still image while MozFest video is loading.

### CODE REVIEW

File: network-api/networkapi/mozfest/templates/fragments/hero/large.html [hunk 1]
```diff
@@ -3,7 +3,8 @@
 <div class="background-wrapper">
   <div class="overlay"></div>
   {% if banner_video_type == "hardcoded" %}
-    <video class="banner-video" playsinline muted loop preload poster="" >
+    <video class="banner-video" playsinline muted loop preload
+           poster="https://s3.amazonaws.com/mofo-assets/foundation/video/mozfest-banner-video-poster.png" >
```
Comment (mmmavis) [inline network-api/networkapi/mozfest/templates/fragments/hero/large.html]: I put the file on s3 as welll so if we ever have to update the video, we can just update both video and the still image without a new deploy.

Comment (Pomax) [reply network-api/networkapi/mozfest/templates/fragments/hero/large.html]: nice!

**Comment by Pomax** [comment]
As "natural" data, we should probably use JPG compression rather than PNG - right now it's a 1.7MB image, which will take a long time to transfer as well, whereas because the kind of image data it is, even a q=3 JPG will still look pretty good at only 155kb:

mozfest-banner-video-poster

**Comment by mmmavis** [comment]
Good call. PR & images have been updated!

**Pull request merged**